src/cromshell/utilities/HTTPUtils.py

```diff
@@ -0,0 +1,27 @@
+import logging
+import sys
+
+import requests
+
+from cromshell.utilities import IOUtils
+
+LOGGER = logging.getLogger(__name__)
+
+
+def assert_can_communicate_with_server(cromshell_url):
+    """Check Connection with Cromwell Server"""
+
+    try:
+        curl_out = requests.get("{}/api/workflows/v1/backends".format(cromshell_url))
+    except ConnectionError:
+        LOGGER.error("Failed to connect to %s", cromshell_url)
+        sys.exit()
```

Comment:
Since there's an error we should exit with some error code here to specify it. 

Traditionally `0` means success and any other value was an error.  You could then look up  those other values in a  table to better understand what the error actually was.

For now  I recommend using `1`: 

```
sys.exit(1)
```